monkeytype/typing.py

```diff
@@ -251,8 +251,19 @@ def rewrite(self, typ):
         return typ
 
 
+class RewriteGenerator(TypeRewriter):
+    """Returns an Iterator, if the send_type and return_type of a Generator is None"""
+
+    def rewrite_Generator(self, typ):
+        args = typ.__args__
+        if issubclass(args[1], NoneType) is True and issubclass(args[2], NoneType) is True:
```

Comment:
this should just be `if args[1] is None and args[2] is None`.

`is True` is redundant and should be omitted in both cases, and `is None` is better than `issubclass(..., NoneType)`

Reply:
Or wait, I see that's what you had before; are these actually `NoneType` and not `None`? if so, you should still omit `is True`, and use `is NoneType` instead of `issubclass(..., NoneType)`